app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb

```diff
@@ -1,8 +1,13 @@
 class ManageIQ::Providers::Autosde::StorageManager::CloudVolume < ::CloudVolume
   supports :create
-  supports :safe_delete
+  supports :safe_delete do
+    unsupported_reason_add(:safe_delete, _("the volume is not connected to an active provider")) unless ext_management_system
+  end
   supports :update do
-    unsupported_reason_add(:update, _("The Volume is not connected to an active Provider")) unless ext_management_system
+    unsupported_reason_add(:update, _("the volume is not connected to an active provider")) unless ext_management_system
+  end
+  supports :delete_volume do
```

Comment:
Think this should be `supports :delete` since that is the common pattern for crud operations (`:create`, `:update`, `:delete`)